browser/src/layer/tile/CommentListSection.ts

```diff
@@ -1992,6 +1992,16 @@ export class CommentSection extends CanvasSectionObject {
 			comment.onCommentDataUpdate();
 		}
 	}
+
+	public rejectAllTrackedCommentChanges(): void {
+		for (var i = 0; i < this.sectionProperties.commentList.length; i++) {
+			var comment = this.sectionProperties.commentList[i];
+			if (comment.sectionProperties.data.layoutStatus === 3) {
+				comment.sectionProperties.data.layoutStatus = 2;
```

Comment:
what are these magic numbers?
best to have them named by some static const in the class

Reply:
it is explained here in the comments: https://github.com/CollaboraOnline/online/blob/71a427c465c3b952f032b283cedc9e52ea985fcd/browser/src/layer/tile/CommentSection.ts#L69

but yes good to have some class